cmd/ocm-backplane/login/login.go

```diff
@@ -183,6 +185,13 @@ func runLogin(cmd *cobra.Command, argv []string) (err error) {
 	}
 
 	logger.Debugf("Using backplane URL: %s\n", bpURL)
+	backplanehost, err := getBackplaneHost(bpURL)
```

Comment:
The backplanehost here is a bit confusing, can we update it to `getBackplaneCNAME` ?
also the variable should use the camel case

Reply:
The function `getBackplaneHost(bpURL) `has been updated to the `getBackplaneCNAME(bpURL) `in the a7d76c7
As per the Go's mixedCase convention,  the variable `backplanehost`   has been renamed to the variable `backplaneResolution` in the commit  a7d76c7